### PR TITLE
Jackie's Adagrams <3

### PR DESCRIPTION
**Pull request by jfahmy**
# JS Adagrams
Congratulations! You're submitting your assignment!

## Comprehension Questions
Question | Answer
--- | ---
What patterns were you able to use from your Ruby knowledge to apply to JavaScript? | I  already had the code written in ruby, using loops and conditionals, so I was able to translate these to javascript easily by referencing the textbook materials from our lectures. 
Did you need to use different strategies to find information online about JavaScript than you would for Ruby? | There is some slightly different vocabulary - like "function" and "callback" vs. method and "object" as opposed to hash, but I think the general strategy is the same.  At one point I needed to Google why my forEach loop was not breaking out of the loop and returning, but I failed at finding the information I needed so I must have not been wording the question correctly. (I just tried googling it now and I was able to find it on the first stackoverflow post, so I'm not sure why I had so much trouble locating that info yesterday.)
What was a challenge you were able to overcome on this assignment? | Unfamiliar syntax and the linter constantly throwing red dots at me. I find the linter a little frustrating because it shows a red dot even when you haven't finished typing - like it will say un-used variable or "empty block statement" - which makes me pause and think I have a syntax error when really I just haven't finished writing my block yet. Overall though, I think the linter will be helpful. 
What is your favorite thing about learning a new programming language? | Everything! It's an adventure and I'm excited by how much I don't know. Also, console.logging all over the place. I enjoy debugging so it's not so bad. 
What is your least favorite thing about learning a new programming language? | Not knowing the terminology so not understanding the correct way to ask my questions (either to a person or to google).
Do you have any recommendations on how we could improve this project for the next cohort? | Perhaps we should write some of the tests? We worked on testing a bit yesterday, but it was not re-enforced by this assignment.

### CODE REVIEW

File: src/adagrams.js [hunk 1]
```diff
@@ -1,8 +1,99 @@
 const Adagrams = {
   drawLetters() {
-    // Implement this method for wave 1
+    let tiles = ["A","A","A","A","A","A","A","A","A","B","B","C","C","D","D","D","D","E","E","E","E","E","E","E",
```
Comment (CheezItMan) [inline src/adagrams.js:3]: Default to `const` instead of `let`.

**Comment by CheezItMan** [comment]
# JS Adagrams
## What We're Looking For

Feature | Feedback
--- | ---
**General** |
Answered comprehension questions |  We'll give you a chance to write some tests this week (a bit), and more after interviews.  There are settings for the linter to only show the red dots, AFTER you save.  
Small commits with meaningful commit messages |  More granular commits would be better, focus on describing functionality rather than tests for commit messages.  
**Code Requirements** |
`drawLetters` method |  Check
Uses appropriate data structure to store the letter distribution |  Check
All tests for `drawLetters` pass |  Check
`usesAvailableLetters` method |  Check
All tests for `usesAvailableLetters` pass |  Check
`scoreWord` method |  Check
Uses appropriate data structure to store the letter scores |  Check
All tests for `scoreWord` pass |  Check
`highestScoreFrom` method |  Check
Appropriately handles edge cases for tie-breaking logic |  Check
All tests for `highestScoreFrom` pass |  Check
**Overall** |  Well done, you hit all the learning goals for the project.  Nicely done, do default to using `const` over `let`, but otherwise you did an excellent job.
